lib/options.json

```diff
@@ -322,6 +322,9 @@
     "public": {
       "type": "string"
     },
+    "setupExitSignals": {
```

Comment:
hm, why we don't do it by default? This option can be very misleading for developers

Reply:
This is the default for the CLI. However, for API this would lead to unexpected behavior, as this may close the process that is using the API. I think that should be up to the user of the API.

Reply:
yep, agree, we will improve docs here